keyboards/late9/rev1/rev1.c

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2021 rookiebwoy
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "rev1.h"
+
+// OLED logo
+#ifdef OLED_ENABLE
+__attribute__((weak)) void oled_task_user(void) {
```

Comment:
```suggestion
bool oled_task_kb(void) {
    if (!oled_task_user()) { return false; }
```

Reply:
I didn't understand why this change...but I trust you lol
Can you explain me, please?

Reply:
the attribute weak was because there was no kb level function, so it was a hack.  There is now (as of nov 27th). And that's the better method.